app/config/logging-cfg-local.yml

```diff
@@ -20,10 +20,6 @@ loggers:
     level: DEBUG
     handlers:
       - console
-  default:
-    level: DEBUG
-    handlers:
-      - console
```

Comment:
Just for curiosity why did you remove this ?

Reply:
I added this some time ago but I think it's not used (since we use `.getLogger(__name__)` everywhere and not a common default logger)